### DISCUSSION TITLE
examples/sample_player.py crashed with "Segmentation fault"

### DISCUSSION THREAD

**Discussion by aweiersmueller** [PaulleDemon/tkVideoPlayer#43] [Q&A]
Hey, 

your example examples/sample_player.py crashed with "Segmentation fault" when:
- a played video is paused
- a new file is loaded

`
pi@raspberrypi:~/$ python videoPlayer.py 
Segmentation fault
`

Thanks for looking into it.

**Comment by PaulleDemon** [comment]
Hi I can't reproduce this error since I don't have a Raspberry with me. Would be more helpful if you can provide full traceback or if you can try and figure out whats going wrong. Also monitor the memory usage while you are at it.
